backend_API/CONTRIBUTING.md

```diff
@@ -0,0 +1,176 @@
+# Contributing Guidelines:
+
+We welcome all contributions to the Open Science Community in Saudi Arabia. The contribution can be an [issue report](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/issues)
+or a [pull request](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/pulls).
+
+## BACKEND API for MOOCs platform
+
+This project is meant to serve as the backend API for the MOOCs platform. The program aims to build an educational platform to serve courses and videos, where each course may have exercises attached.
+
+The API is hosted for testing at https://moocs-test.onrender.com/
+
+#### API DOCUMENTATION
+Documentation for consuming the API endpoints can be found here https://documenter.getpostman.com/view/20633788/2s8YemuZwv
+
+#### Technologies
+
+- Node JS - Backend Environment
+- Express JS - Framework for APIs
+- Mongo DB - Non Relational Database
+- Mocha and Chai - Framework for Unit tests
+
+### CONTRIBUTING WORKFLOW
+
+1. Check that there isn't [already an issue](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/issues) about your idea to avoid duplicating work.
+
+   - If there isn't one already, please create one so that others know you're working on it.
+
+2. Fork the [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/) to your GitHub account.
+
+3. Clone the forked repository on your local machine.
+
+```bash
+
+git clone https://github.com/<your-github-username>/Open-Science-Community-in-Saudi.git
+
+```
+
+4. Sync the fork, to avoid merge conflicts.
+
+```bash
+
+    git remote add upstream https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi.git
+    git fetch upstream
+    git pull upstream master
+    git push
+
+```
+
+5. From the dev_team2 branch, create a new branch with your github username as its name.
+
+```bash
+   git checkout dev_team2
+   git checkout -b <commit-message>
+
+```
+Your commit message should be descriptive enough, an exmaple is `feat-signpup` or `fix-failed-login`
```

Comment:
Line 53:  git checkout -b <branch-name>

Line 56:
Commit messages are more descriptive Examples: Added signup page, Fixed typoerror in landing page, 
https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

Reply:
sorry, i made a mistake, i was thinking it was for branch name

Reply:
The line was for branch messages, not commit messages, i just made the correction

Reply:
👍